### PR TITLE
Moved tests out of run_in_graph_and_eager_mode in sparse_image_warp_test

### PR DESCRIPTION
**Pull request by autoih**
cc @gabrieldemarmiesse

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
It seems there are some conflicts. Could you fix them? Thanks!

File: tensorflow_addons/image/sparse_image_warp_test.py [hunk 1]
```diff
@@ -26,75 +26,32 @@
...
-        regularization_weight=regularization,
-        num_boundary_points=num_boundary_points,
-    )
+def test_get_boundary_locations():
+    image_height = 11
+    image_width = 11
+    num_points_per_edge = 4
+    locs = _get_boundary_locations(image_height, image_width, num_points_per_edge)
+    num_points = locs.shape[0]
+    np.testing.assert_equal(num_points, 4 + 4 * num_points_per_edge)
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/image/sparse_image_warp_test.py]: `assert ... == ...` should be enough.

File: tensorflow_addons/image/sparse_image_warp_test.py [hunk 2]
```diff
@@ -26,75 +26,32 @@
...
+    locs = [(locs[i, 0], locs[i, 1]) for i in range(num_points)]
+    for i in (0, image_height - 1):
+        for j in (0, image_width - 1):
+            assert (i, j) in locs, "{},{} not in the locations".format(i, j)
+
+        for i in (2, 4, 6, 8):
+            for j in (0, image_width - 1):
+                assert (i, j) in locs, "{},{} not in the locations".format(i, j)
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/image/sparse_image_warp_test.py]: You can remove the error message. The one provided by pytest will have more info.

Comment (autoih) [reply tensorflow_addons/image/sparse_image_warp_test.py]: Thanks @gabrieldemarmiesse.

File: tensorflow_addons/image/sparse_image_warp_test.py [hunk 3]
```diff
@@ -66,35 +66,32 @@ def assert_zero_shift(order, regularization, num_boundary_points):
...
+        for j in (0, image_width - 1):
+            assert (i, j) in locs, "{},{} not in the locations".format(i, j)
+
+        for i in (2, 4, 6, 8):
+            for j in (0, image_width - 1):
+                assert (i, j) in locs
+
+        for i in (0, image_height - 1):
+            for j in (2, 4, 6, 8):
+                assert (i, j) in locs, "{},{} not in the locations".format(i, j)
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/image/sparse_image_warp_test.py]: We don't need the error message anymore, the one provided by pytest is enough.

File: tensorflow_addons/image/sparse_image_warp_test.py [hunk 4]
```diff
@@ -66,35 +66,32 @@ def assert_zero_shift(order, regularization, num_boundary_points):
...
+    image_height = 11
+    image_width = 11
+    num_points_per_edge = 4
+    locs = _get_boundary_locations(image_height, image_width, num_points_per_edge)
+    num_points = locs.shape[0]
+    assert num_points == (4 + 4 * num_points_per_edge)
+    locs = [(locs[i, 0], locs[i, 1]) for i in range(num_points)]
+    for i in (0, image_height - 1):
+        for j in (0, image_width - 1):
+            assert (i, j) in locs, "{},{} not in the locations".format(i, j)
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/image/sparse_image_warp_test.py]: Same here

**Pull request merged**